cilium/cmd/bpf_lb_list.go

```diff
@@ -69,22 +69,23 @@ func dumpSVC(serviceList map[string][]string) {
 		svc := svcKey.String()
 		revNATID := svcVal.GetRevNat()
 		backendID := svcVal.GetBackendID()
+		flags := svcVal.GetFlags()
 
 		if backendID == 0 {
 			ip := "0.0.0.0"
 			if svcKey.IsIPv6() {
 				ip = "[::]"
 			}
-			entry = fmt.Sprintf("%s:%d (%d)", ip, 0, revNATID)
+			entry = fmt.Sprintf("%s:%d (%d) [FLAGS: 0x%x]", ip, 0, revNATID, flags)
```

Comment:
I would prefer if we add actual human readable names instead of hex dump, eases debugging, also for the other places.